src/main/java/baseball/io/PlayerIoManager.java

```diff
@@ -0,0 +1,51 @@
+package baseball.io;
+
+import baseball.constant.NumberConstant;
+import baseball.io.console.ConsoleReader;
+import baseball.io.console.ConsoleWriter;
+import baseball.model.judge.JudgeResult;
+import baseball.utils.NumberValidator;
+import java.util.List;
+
+public class PlayerIoManager {
+
+    private ConsoleReader consoleReader;
+    private ConsoleWriter consoleWriter;
+
+    private PlayerIoManager(ConsoleReader consoleReader, ConsoleWriter consoleWriter) {
+        this.consoleReader = consoleReader;
+        this.consoleWriter = consoleWriter;
+    }
+
+    public static PlayerIoManager create() {
+        return new PlayerIoManager(new ConsoleReader(), new ConsoleWriter());
+    }
```

Comment:
이 부분도 DI를 활용하면 이렇게 바꿀 수 있겠네요!
```suggestion
  public PlayerIoManager() {
          this.consoleReader = new ConsoleReader();
          this.consoleWriter = new ConsoleWriter();
      }
```